src/database/models/project_item.rs

```diff
@@ -959,7 +1070,7 @@ pub struct QueryProject {
     pub inner: Project,
     pub categories: Vec<String>,
     pub additional_categories: Vec<String>,
-    pub versions: Vec<VersionId>,
+    pub public_versions: Vec<VersionId>,
```

Comment:
why still have this attr on query project? isn't the point of this PR to remove it?